lab_6_pipeline/pipeline.py

```diff
@@ -62,11 +114,21 @@ def __init__(
             corpus_manager (CorpusManager): CorpusManager instance
             analyzer (LibraryWrapper | None): Analyzer instance
         """
+        self._corpus = corpus_manager
+        self._analyzer = analyzer
 
     def run(self) -> None:
         """
         Perform basic preprocessing and write processed text to files.
         """
+        docs = self._analyzer.analyze([article.text for article in
+                                       self._corpus.get_articles().values()])
+
+        for i, article in enumerate(self._corpus.get_articles().values()):
```

Comment:
consider giving it a longer name, one-character variable names are bad code style

Reply:
fixed